src/alfasim_score/constants.py

```diff
@@ -1,5 +1,16 @@
+from barril.units import Scalar
+
+from alfasim_score.units import ROUGHNESS_UNIT
+
 WELLBORE_TOP_NODE = "WELBORE_TOP_NODE"
 WELLBORE_BOTTOM_NODE = "WELBORE_BOTTOM_NODE"
 ANNULUS_TOP_NODE_NAME = "WELLBORE_ANNULUS_TOP_NODE"
 
 CEMENT_NAME = "cement"
+
+ROCK_DEFAULT_ROUGHNESS = Scalar(0.1, ROUGHNESS_UNIT)
```

Comment:
Are these values the same we are using in PWPa and ALFAtwin?

Reply:
For the ALFAtwin and PWPa use the roughness configured in the model, so I used here the same default values I previously configured as default for floweditio.